src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/AdeccoSteps.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.ons.fsdr.tests.acceptance.steps;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.PropertySource;
+import uk.gov.ons.fsdr.common.dto.AdeccoResponse;
+import uk.gov.ons.fsdr.common.dto.AdeccoResponseJob;
+import uk.gov.ons.fsdr.common.dto.AdeccoResponseWorker;
+import uk.gov.ons.fsdr.tests.acceptance.utils.AdeccoPeopleFactory;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@PropertySource("classpath:application.properties")
+public class AdeccoSteps {
+
+  public static AdeccoResponse adeccoResponse = new AdeccoResponse();
+  public static List<AdeccoResponse> adeccoResponseList = new ArrayList<>();
+
+  @Given("An employee exists in {string} with an id of {string}")
+  public void we_recieve_an_employee_with_an_id_of(String source, String id) {
```

Comment:
source isn't used

Reply:
Actually, I just saw the feature file, leave the **source** there.